swagger.yaml

```diff
@@ -146,226 +328,821 @@ paths:
+      parameters:
+      - $ref: '#/parameters/instance_id'
+      produces:
+      - "application/json"
+      security:
+      - FlorenceAPIKey: []
+      responses:
+        200:
+          description: "Return a single instance state"
+          schema:
+            $ref: '#/definitions/Instance'
+        404:
+          description: "InstanceId does not match any import jobs"
+        500:
+          $ref: '#/responses/InternalError'
+    put:
+      tags:
+      - "Private"
+      summary: "Update an instances properties"
+      description: "Update an instance by providing an unique id and a set of properties to over write"
+      parameters:
+      - $ref: '#/parameters/instance_id'
+      - $ref: '#/parameters/instance'
+      produces:
+      - "application/json"
+      security:
+      - InternalAPIKey: []
+      responses:
+        200:
+          description: "The instance has been updated"
+        404:
+          description: "InstanceId does not match any import jobs"
+        500:
+          $ref: '#/responses/InternalError'
+  /instances/{instance_id}/dimensions:
+    get:
+      tags:
+      - "Private user"
+      summary: "Get all dimensions from an instance"
+      description: "Get all dimensions from an instance"
+      parameters:
+      - $ref: '#/parameters/instance_id'
+      produces:
+      - "application/json"
+      security:
+      - InternalAPIKey: []
+      responses:
+        200:
+          description: "Return a list of dimensions"
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/DimensionNode'
+        404:
+          description: "InstanceId does not match any instances"
+        500:
+          $ref: '#/responses/InternalError'
+  /instances/{instance_id}/dimensions/{dimension}/options:
+    get:
+      tags:
+      - "Private user"
+      summary: "Get all unique values from a dimension"
+      description: "Get all unique values from a dimension"
+      parameters:
+      - $ref: '#/parameters/instance_id'
+      - $ref: '#/parameters/dimension'
+      produces:
+      - "application/json"
+      security:
+      - InternalAPIKey: []
+      responses:
+        200:
+          description: "Return a list of unique values"
+          schema:
+            type: object
+            properties:
+              dimension_id:
+                description: "The name of the dimension"
+                type: string
+              values:
+                description: "A list of all unique values within a dimension"
+                type: array
+                items:
+                  type: string
+        400:
+          description: "instance_id does not match any instances"
+        404:
+          description: "dimension_id does not match any dimensions within the instance"
+        500:
+          $ref: '#/responses/InternalError'
+    put:
+      tags:
+      - "Private"
+      summary: "Update the number of observations within this instance"
```

Comment:
Something seems not right here. The PUT is for the path `/instances/{instance_id}/dimensions/{dimension}/options` but the description is related to the number of observations. I am guessing the path is incorrect?

Reply:
This endpoint was a mistake and it shouldn't be here